docs/pages/review/share-previews-with-your-team.mdx

```diff
@@ -0,0 +1,34 @@
+---
+title: Share previews with your team
+description: Share previews of your changes with your team by publishing updates on branches.
```

Comment:
In the PR description, it is mentioned that we want to recommend "channels" instead of "branches". Should we also apply that context to this guide? We also use this term as if the user is already familiar. May we should link to the first instance of a branch here: https://docs.expo.dev/eas-update/eas-cli/?

Reply:
There are two main use cases for updates. One is for sending production updates to end-users. For those users, I think we should only mention `--channel`, and not mention how branches work with channels until a more advanced doc in a guide.

The other use case is for developers to publish preview updates on every git branch they create. In this doc, I am referring to git branches. One thing I am papering over is that when you run `eas update --auto`, we make an update branch on our servers, so that we can provide the preview update. That detail is not important to get the value from this feature.

So, in general, if we can write docs such that new users don't need to know how the system works to use it, that's ideal. With that, later in advanced docs, we absolutely should explain how update works on our servers so that developers can create custom/complex workflows if they require it.

Reply:
Thanks for the explanation! I thought the branches referenced here were Update branches 😄